components/src/dynamo/vllm/handlers.py

```
@@ -625,6 +625,261 @@ async def wake_up(self, body: dict) -> dict:
+                        logger.info(
+                            f"[RL] Published LoRA '{lora_name}' ModelDeploymentCard"
+                        )
+                    except Exception as e:
+                        # Rollback: remove the LoRA from the engine to keep state consistent.
+                        logger.exception(
+                            f"[RL] Failed to publish LoRA '{lora_name}' MDC: {e}; rolling back add_lora"
+                        )
+                        try:
+                            await self.engine_client.remove_lora(lora_id)
+                        except Exception:
+                            pass
```

Ruff (0.15.11)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)